spec/models/user_spec.rb

```diff
@@ -1,14 +1,16 @@
 require 'rails_helper'
 
 describe User do
-  # subject { FactoryGirl.build(:user) }
+  subject { FactoryGirl.build(:user) }
+  it { should have_many(:adventures).through(:adventure_users) }
   it { should have_many(:bucket_lists) }
 
-  # it { should have_valid(:username).when('useruser', 'iamauser12345') }
-  # it { should_not have_valid(:username).when(nil, '') }
-  #
-  # it { should validate_uniqueness_of(:username) }
-  #
-  # it { should have_valid(:email).when('user@mailhere.com', 'thisuser@mail.com') }
-  # it { should_not have_valid(:email).when(nil, '', 'user', '@', 'usermail.com', 'user@mail') }
+  it { should have_valid(:username).when("useruser", "iamauser12345") }
+  it { should_not have_valid(:username).when(nil, " ") }
+
+  it { should validate_uniqueness_of(:username) }
+
+  it { should have_valid(:email).when("user@mailhere.com", "thisuser@mail.com") }
+  it { should_not have_valid(:email)
+    .when(nil, " ", "user", "@", "usermail.com", "user@mail") }
```

Comment:
Expression at 15, 63 should be on its own line.<br>Place the . on the previous line, together with the method call receiver.